src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md

```diff
@@ -1,17 +1,53 @@
-# OpenTelemetry.Instrumentation.GrpcNetClient
+# Grpc.Net.Client Instrumentation for OpenTelemetry
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 
-Automatically instruments the outgoing requests from
-[Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client).
+This is an [Instrumentation Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumentation-library)
+which instruments [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client)
+and collects telemetry about outgoing gRPC requests.
 
-## Installation
+## Steps to enable OpenTelemetry.Instrumentation.GrpcNetClient
+
+### Step 1: Install Package
+
+Add a reference to the
+[`OpenTelemetry.Instrumentation.GrpcNetClient`](https://www.nuget.org/packages/opentelemetry.instrumentation.grpcnetclient)
+package. Also, add any other instrumentations & exporters you will need.
 
 ```shell
 dotnet add package OpenTelemetry.Instrumentation.GrpcNetClient
 ```
 
+### Step 2: Enable Grpc.Net.Client Instrumentation at application startup
+
+Grpc.Net.Client instrumentation must be enabled in your code. The following
+example demonstrates how to do this for an ASP.NET Core application. The
+example also sets up the OpenTelemetry Console exporter and ASP.NET Core
```

Comment:
Asp.Net Core intrumentation is not required to demonsteate the Grpc usage, so lets remove it?

Reply:
The reason I put this in is that the library requires there be a parent activity in order for it to generate an activity for the gRPC invocation https://github.com/grpc/grpc-dotnet/blob/ef77760676ce19c279d5480bca68af3ea4d60e96/src/Grpc.Net.Client/Internal/GrpcCall.cs#L680-L683.

Reply:
When we enable diagnosticlistener, then also it should start activity. (Its a set of "OR" conditions, and one of them is parent activity, another is enablediagnosticlistener).
Do you observer other behaviour?

Reply:
Ah yes, sorry, that's not the code I was thinking of.. I _think_ I've observed behavior otherwise, though give me a sec, confirming I'm not just making things up...

Reply:
I do observe the behavior. I mistakingly thought this was due to how the library was instrumented. The behavior is not specific to Grpc.Net.Client instrumentation. I've opened #1217 demonstrating it with HttpClient instrumentation as well.